components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/ExceptionCodes.java

```diff
@@ -416,8 +416,9 @@
     ERROR_READING_PARAMS_FILE(900901, "Error while reading meta information from the api_params.yaml file", 400,
             "Error while reading meta information from the api_params.yaml file"),
     NO_API_ARTIFACT_FOUND(900902, "No Api artifacts found for given criteria", 404,
-            "No Api artifacts found for given criteria");
-
+            "No Api artifacts found for given criteria"),
+    GATEWAY_TYPE_NOT_FOUND(900903, "Gateway Type not found", 404, "Gateway Type not found available Gateway Types : " +
```

Comment:
Type should be 'type' :) . Lets not use capital letters in middle of the sentense